completions/functions.py

```diff
@@ -4,6 +4,16 @@
 # This dict maps function names to their completions. It includes pseudo-class
 # and pseudo-element functions like :nth-child() and ::attr().
 func_name_to_completions = {
+    "
+    "clamp": [
+        ("length",),
+        ("frequency",),
+        ("angle",),
+        ("time",),
+        ("percentage",),
+        ("number",),
+        ("integer",),
+    ],
```

Comment:
[I've got types for these](https://github.com/ryboe/CSS3/blob/master/completions/types.py). `types.py` is intended to reduce some of the redundancy, since there are so many functions that take lengths, percentages, etc. `t.length` will offer `calc()` as a completion option as well.

```py
"clamp": (
    t.length
    + t.frequency
    + t.angle
    + t.time
    + t.percentage
    + t.number
    + t.integer
)
```